code/modules/economy/EFTPOS.dm

```diff
@@ -134,9 +115,9 @@
 			if("change_id")
 				var/attempt_code = text2num(input("Re-enter the current EFTPOS access code", "Confirm EFTPOS code"))
 				if(attempt_code == access_code)
-					var name = input("Enter a new terminal ID for this device", "Enter new EFTPOS ID") as text|null
+					var/name = input("Enter a new terminal ID for this device", "Enter new EFTPOS ID") as text|null
 					if(name)
-						eftpos_name = name + " EFTPOS scanner"
+						eftpos_name = "[name] EFTPOS scanner"
```

Comment:
Huh, one would have thought this would have changed the actual thing's terminal ID. Eh, well, I'm not going to lump a fix request onto this PR, as this is how it was behaving before